src/api/goods.ts

```diff
@@ -10,10 +10,12 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => goods
+      .sort((a: Good, b: Good) => a.name.localeCompare(b.name)).slice(0, 5));
 };
 
-export const getRedGoods = () => {
+export const getRed = () => {
   return getAll()
-    .then(goods => goods); // get only red
+    .then(goods => goods
+      .filter(good => good.color === 'red'));
```

Comment:
you need only color here
```suggestion
      .filter(({ color }) => color === 'red'));
```